client/src/components/Profile/Drawer.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { Drawer as MUIDrawer } from '@material-ui/core';
+import { NavLink } from 'react-router-dom';
+import { makeStyles } from '@material-ui/styles';
+import profilePic from '../../static/images/profilePicSample.png';
+// import Avatar from '@material-ui/core/Avatar';
+
+const useStyles = makeStyles(theme => ({
+    linkStyles: {
+        textDecoration: 'none',
+        color: 'black',
+        padding: '1em 0 0 2em',
+    },
+    profilePic: {
+        borderRadius: '50%',
+        width: '130px',
+        marginTop: '2em',
+    },
+    drawer: {
+        width: 400,
+    },
+    linkContainer: {
+        paddingTop: '2em',
+    },
+    profileName: {
+        paddingTop: '1em',
+        fontSize: 22,
+    },
+    test: {
+        background: '#F9F9FC',
+    },
+}));
+
+const Drawer = props => {
+    const classes = useStyles();
+
+    return (
+        <MUIDrawer variant="permanent" anchor="left" className={classes.drawer}>
+            {/* Drawer container */}
+            <Grid
+                container
+                direction="column"
+                alignItems="center"
+                style={{ width: '400px' }}
```

Comment:
I would avoid combining inline styling with makeStyles.  Move this to the `test` class (maybe rename `test` to be less.. testy 😆 )